web/components/visibility-observer.tsx

```diff
@@ -8,17 +8,18 @@ export function VisibilityObserver(props: {
   const { className } = props
   const [elem, setElem] = useState<HTMLElement | null>(null)
   const onVisibilityUpdated = useEvent(props.onVisibilityUpdated)
-
-  useEffect(() => {
-    const hasIOSupport = !!window.IntersectionObserver
```

Comment:
https://caniuse.com/intersectionobserver

Reply:
I guess we don't support IE11 or old versions of other browsers.